jobs/openshift/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,28 @@
+#!groovy
+
+def clusterName = params.clusterName
+
+currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+
+node('cirhos_rhel7') {
+    cleanWs()
+    stage('Create OpenShift Cluster') {
+        println("Create OpenShift Cluster")
+
+        wrap([$class: 'AnsiColorBuildWrapper', colorMapName: "xterm"]) {
+            ansibleTower(
+                    towerServer: 'Dev Tower',
+                    jobTemplate: '[aws.eng] Cluster Create',
+                    templateType: 'workflow',
+                    importTowerLogs: true,
+                    removeColor: false,
+                    verbose: true,
+                    extraVars: """---
+                    oo_clusterid: ${clusterName}
```

Comment:
Does this generate an id if it's an empty string?

Maybe we should set a default value for `clusterName` in case the param isn't defined.

```
def clusterName = params.clusterName ?: ''
```

Reply:
The default value is an empty string, so no need to default it like that. That said, it should probably check for an empty string and not continue if it is, but we should probably also be doing that in lots of jobs.

Reply:
I added an input verification stage https://github.com/integr8ly/ci-cd/pull/95/commits/21d817e67e2ecf152ec139d0fd30c4742518a1c0, this is more like what we should be doing.